client/layout/sidebar/style.scss

```diff
@@ -132,6 +132,22 @@
 		.sidebar__menu-icon {
 			fill: var( --color-sidebar-menu-hover-text );
 		}
+
+		.count {
+			border-color: var( --color-sidebar-menu-hover-text );
+		}
+	}
+
+	.count {
+		margin-left: 8px;
+		color: var( --color-sidebar-menu-hover-text );
+		background-color: var( --color-sidebar-menu-hover-background );
+		border-color: var( --color-sidebar-menu-hover-background );
+		line-height: 0;
+		margin-left: 8px;
+		padding: 8px 6px;
```

Comment:
@jancavan Can those be removed? Line-height and padding seem to be set from the original component and margin-left is set a few lines further up

```suggestion
```

Reply:
@obenland Hmm, those were added to fix the additional padding issue. It looks like its default line-height was creating additional space, but I'm open to removing them if there's an alternative fix. Would an `is-modifier` containing those properties work for the `count` that's specifically on the sidebar? Open to suggestions.

Reply:
Removed extra `margin-left`

Reply:
Sounds good. Let's get this in! 🙂